tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -282,9 +267,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             fixtures: {
                 type: "fluid.test.testCaseHolder",
                 options: {
```

Comment:
I was referring to the configuration specified in this options block. You can pull all of that out into a separate grade and just set the type to the new grade.